wwwapp/views.py

```diff
@@ -324,18 +326,30 @@ def participants_view(request, year):
             if participant.participant.user.id in lecturers_ids:
                 continue
 
+            birth = participant.participant.user_info.pesel[:6]
+            is_adult = None
+            if birth is not None:
+                if not birth.isdigit() or len(birth) != 6:
+                    birth = None
+                else:
+                    birth = datetime.date(int(birth[:2]) + 2000, int(birth[2:4]), int(birth[4:6]))
```

Comment:
**Does not work at all.** :joy:
![image](https://user-images.githubusercontent.com/1517255/61124963-bcc50400-a4a8-11e9-8ddf-c46137590f0a.png)